### PR TITLE
[RAPPS] Remove string casts after the conutils fix

### PR DESCRIPTION
**Pull request by learn-more**

### CODE REVIEW

**Review by HBelusca — APPROVED** [review]

File: base/applications/rapps/unattended.cpp [hunk 1]
```diff
@@ -228,12 +225,12 @@ BOOL HandleHelpCommand(LPWSTR szCommand, int argcLeft, LPWSTR * argvLeft)
...
 
-    ConPrintf(StdOut, (LPWSTR)L"\n");
+    ConPrintf(StdOut, L"\n");
     ConResPuts(StdOut, IDS_APPTITLE);
-    ConPrintf(StdOut, (LPWSTR)L"\n\n");
+    ConPrintf(StdOut, L"\n\n");
 
     ConResPuts(StdOut, IDS_CMD_USAGE);
-    ConPrintf(StdOut, (LPWSTR)L"%ls\n", UsageString);
+    ConPrintf(StdOut,L"%ls\n", UsageString);
```
Comment (HBelusca) [inline base/applications/rapps/unattended.cpp:233]:
```suggestion
    ConPrintf(StdOut, L"%ls\n", UsageString);
```

File: base/applications/rapps/unattended.cpp [hunk 2]
```diff
@@ -125,7 +122,7 @@ BOOL CALLBACK CmdFindAppEnum(CAvailableApplicationInfo *Info, BOOL bInitialCheck
         return TRUE;
     }
 
-    ConPrintf(StdOut, (LPWSTR)L"%s (%s)\n", (LPCWSTR)(Info->m_szName), (LPCWSTR)(Info->m_szPkgName));
+    ConPrintf(StdOut, L"%s (%s)\n", (LPCWSTR)(Info->m_szName), (LPCWSTR)(Info->m_szPkgName));
```
Comment (HBelusca) [inline base/applications/rapps/unattended.cpp:125]: Are the casts for `Info->m_szName` and co. still necessary? (I don't think so)

Comment (learn-more) [reply base/applications/rapps/unattended.cpp:125]: yes they are, since those are CString's

Comment (learn-more) [reply base/applications/rapps/unattended.cpp:125]: I'll change that to `.GetString()` to be a bit more obvious

**Pull request merged**